framework/db/oci/QueryBuilder.php

```diff
@@ -162,6 +162,27 @@ public function addForeignKey($name, $table, $columns, $refTable, $refColumns, $
     }
 
     /**
+     * For each return column a param will be added to $returnParams.
+     * They should have variables bound before executing the query.
+     * @inheritdoc
+     */
+    public function insertReturning($table, $columns, &$params, $returnColumns = null, &$returnParams = null)
+    {
+        $schema = $this->db->getSchema();
```

Comment:
I was looking for a way to simplify this too.

First, now since I've moved this to a separate method we don't have to worry that `Command::insert()` calls `bindValues`.

But how a bound param (using `bindParam`) can be returned in `queryOne()`? I actually tried to run in PostgreSQL `INSERT INTO RETURNING col INTO :param` yesterday but it doesn't work as sql, it's only valid for plpgsql.